src/calendar.js

```diff
@@ -38,9 +38,12 @@ class RemoteCalendar {
     this.url = url
     this.headers = {
       Accept: 'application/vnd.opentimestamps.v1',
-      'User-Agent': 'javascript-opentimestamps',
       'Content-Type': 'application/x-www-form-urlencoded'
     }
+    if (!process.browser) { // running in node.js
+      console.log('Setting User-Agent in request header')
```

Comment:
the standard behaviour is to run ots library in nodejs with the right "User-Agent", so we don't need to provide any output. 

If you want to print a message only when you are in the browser, I suggest to revert the condition and instead of insert the user-agent, you could remove it with (keeping the previous headers): `delete this.headers['User-Agent']`

Reply:
removed unnecessary output message